apps/server/src/modules/school/domain/do/school.ts

```diff
@@ -19,25 +25,55 @@ export class School extends DomainObject<SchoolProps> {
 			id: this.props.id,
 			name: this.props.name,
 			language: this.props.language,
-			logo_dataUrl: this.props.logo_dataUrl,
-			logo_name: this.props.logo_name,
+			logo: this.props.logo,
 		};
 
 		return info;
 	}
 
-	public addFeature(feature: SchoolFeature): void {
-		this.props.features.add(feature);
+	public getPermissions(): SchoolPermissions | undefined {
+		const { permissions } = this.props;
+
+		return permissions;
 	}
 
-	public removeFeature(feature: SchoolFeature): void {
-		this.props.features.delete(feature);
+	public addInstanceFeature(feature: InstanceFeature): void {
+		if (!this.props.instanceFeatures) {
+			this.props.instanceFeatures = new Set();
+		}
+		this.props.instanceFeatures.add(feature);
 	}
 
-	public getPermissions(): SchoolPermissions | undefined {
-		const { permissions } = this.props;
+	public removeInstanceFeature(feature: InstanceFeature): void {
+		if (!this.props.instanceFeatures) {
+			return;
```

Comment:
Die Zeile ist nicht getestet, siehe Coverage Report.

Ich fänds übrigens einfacher, es so zu schreiben. Ist zwar Geschmackssache, aber ich glaube, wir hatten mal gesagt, dass wir mehrere returns in einer Funktion vermeiden wollen.
```
if (this.props.instanceFeatures) {
	this.props.instanceFeatures.delete(feature);
}
```